### ISSUE TITLE
findAll

### ISSUE THREAD

**Issue by Yelinz** [ef4/ember-resource-metadata#13]
This addon doesn't support meta attributes retrieved by the `findAll` function.
Is this intentional?

**Comment by ef4** [comment]
Nope, I'm happy to merge a PR adding it. Should be straightforward.
